src/dispatch/auth/service.py

```diff
@@ -79,6 +79,11 @@ def get_current_user(*, db_session: Session = Depends(get_db), request: Request)
         log.debug("No authentication provider. Default user will be used")
         user_email = DISPATCH_AUTHENTICATION_DEFAULT_USER
 
+    if not user_email:
+        log.exception(
+            f"Unable to determine user email based on configured auth provider. Provider: {DISPATCH_AUTHENTICATION_PROVIDER_SLUG}"
```

Comment:
Adding verbiage in case we tried to get the user email from the `DISPATCH_AUTHENTICATION_DEFAULT_USER` env var and this was not defined.
```suggestion
            f"Unable to determine user email based on configured auth provider or no default auth user email defined. Provider: {DISPATCH_AUTHENTICATION_PROVIDER_SLUG}"
```